### PR TITLE
kubevirt, presubmits: Add psa-sig-network lane

### PR DESCRIPTION
**Pull request by RamLavi**
This PR adds a new lane `pull-kubevirt-e2e-k8s-1.24-psa-sig-network`

### CODE REVIEW

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml [hunk 1]
```diff
@@ -2120,6 +2120,45 @@ presubmits:
...
+      testgrid-dashboards: kubevirt-presubmits
+    cluster: prow-workloads
+    decorate: true
+    decoration_config:
+      grace_period: 5m0s
+      timeout: 4h0m0s
+    labels:
+      preset-bazel-cache: "true"
+      preset-bazel-unnested: "true"
+      preset-dind-enabled: "true"
```
Comment (xpivarc) [inline github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:2135]: Please use `preset-podman-in-container-enabled: "true"`. Also if you don't mind can you replace this for other -psa jobs in a separate commit?

**Review by xpivarc — CHANGES_REQUESTED** [review]

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml (hunk 1 above)
Comment (RamLavi) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:2135]: Done

**Review by xpivarc — COMMENTED** [review]
/lgtm
@dhiller 
Can you have a quick look, please?

**Review by dhiller — APPROVED** [review]
/approve

Please take into account that @xpivarc is working on changing PSA enablement in [this PR](https://github.com/kubevirt/kubevirtci/pull/915)

**Review by dhiller — APPROVED** [review]
/approve

**Pull request merged**